client/landing/jetpack-cloud/components/security-icon/index.tsx

```diff
@@ -0,0 +1,40 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import classnames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import okayIcon from './images/security-okay.svg';
+import errorIcon from './images/security-error.svg';
+
+interface Props {
+	icon: string;
+	className?: string;
+}
+
+function SecurityIcon( props: Props ) {
+	const { icon, className } = props;
+
+	let iconPath = okayIcon;
+	if ( 'error' === icon ) {
+		iconPath = errorIcon;
+	}
+
+	return (
+		<img
+			src={ iconPath }
+			className={ classnames( 'security-icon', `security-icon-${ icon }`, className ) }
+			role="presentation"
+			alt=""
```

Comment:
Is this expected to be empty?

Reply:
Yes. Presentational components are allowed to have an empty `alt`, as per[ W3C guidelines](https://www.w3.org/WAI/tutorials/images/decorative/). This was component was copied and modified from `@automattic/components/product-icon`, for context.

Reply:
That's actually really good to know. I've put useless `alt` text in before just as filler, but now that I'm reflecting on it I bet that is annoying for screen readers .

Reply:
One thing I've been noting a lot of the markup in Calypso isn't entirely semantic or friendly. A danger of React is that it abstracts a lot of the HTML, so we don't necessarily produce stuff that's friendly. I'd be interested at some point in trying to purchase a site with a screen reader to see what happens...